src/Reddit/Reddit.ts

```diff
@@ -1,427 +1,644 @@
-import { ExecutionContext } from "ava";
+import { ExecutionContext } from 'ava'
+import { setStyles, selectAll, select } from '../helpers'
 
 /*
  * LipSurf plugin for Reddit.com
  */
 /// <reference types="@lipsurf/types/extension"/>
-declare const PluginBase: IPluginBase;
+declare const PluginBase: IPluginBase
 
-const thingAttr = `${PluginBase.util.getNoCollisionUniqueAttr()}-thing`;
-const COMMENTS_REGX = /reddit.com\/r\/[^\/]*\/comments\//;
+type Maybe<T> = T | null
 
-function thingAtIndex(i: number) {
-  return `#siteTable>div.thing[${thingAttr}="${i}"]`;
+const thingAttr = `${ PluginBase.util.getNoCollisionUniqueAttr() }-thing`
+const COMMENTS_REGX = /reddit.com\/r\/[^\/]*\/comments\//
+const isOldReddit = /https:\/\/old/.test(window.location.href)
```

Comment:
this test needs to be in `init()`. You can give it a default though, of false. Plugin code that runs in the global scope has undefined behavior for LipSurf.